traffic_ops/app/db/seeds.sql

```diff
@@ -97,6 +103,23 @@ FROM public.role
 WHERE "name" in ('operations', 'read-only', 'portal', 'federation', 'steering')
 ON CONFLICT DO NOTHING;
 
+INSERT INTO public.role_capability
+SELECT id, perm FROM public.role
+CROSS JOIN (
+    VALUES ('DNS-SEC:READ'), ('DNS-SEC:DELETE')
+) AS perms(perm)
+WHERE "priv_level" < 20 AND "priv_level" > 0
+    ON CONFLICT DO NOTHING;
```

Comment:
This grants `DNS-SEC:READ` and `DNS-SEC:DELETE` to a bunch of roles that did not have it before: trouter (`1`), read-only (`4`), portal (`6`), steering (`7`), and federation (`8`):

```diff
--- apache.sql	2023-08-31 17:46:13.333267263 -0600
+++ 7764.sql	2023-08-31 17:46:46.260625355 -0600
@@ -3022,6 +3022,16 @@
 6	DELIVERY-SERVICE-SAFE:UPDATE	A_TIMESTAMP
 7	DELIVERY-SERVICE-SAFE:UPDATE	A_TIMESTAMP
 8	DELIVERY-SERVICE-SAFE:UPDATE	A_TIMESTAMP
+1	DNS-SEC:DELETE	A_TIMESTAMP
+4	DNS-SEC:READ	A_TIMESTAMP
+4	DNS-SEC:DELETE	A_TIMESTAMP
+6	DNS-SEC:READ	A_TIMESTAMP
+6	DNS-SEC:DELETE	A_TIMESTAMP
+7	DNS-SEC:READ	A_TIMESTAMP
+7	DNS-SEC:DELETE	A_TIMESTAMP
+8	DNS-SEC:READ	A_TIMESTAMP
+8	DNS-SEC:DELETE	A_TIMESTAMP
+1	DELIVERY-SERVICE-SAFE:UPDATE	A_TIMESTAMP
 3	ACME:READ	A_TIMESTAMP
 4	ACME:READ	A_TIMESTAMP
 6	ACME:READ	A_TIMESTAMP
```

Reply:
I increased the check from 0 to 8.. not sure if that is the right way but hopefully it helps.